### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JEuler**
Added note about Kotlin and changed compile to implementation. 

PS. 
Thanks again for the nice library!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +108,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.github.livefront:bridge:v1.1.3'
+    implementation 'com.github.livefront:bridge:v1.1.3'
```
Comment (byencho) [inline README.md:98]: 👍

**Comment by byencho** [comment]
Thanks for the updates, @JEuler ! So I'm cool with the `compile` to `implementation` change but I'm a bit more cautious on adding Kotlin examples to the `README` (though we do use it for the sample app). My thinking here is that it's pretty standard to use Java for examples in a project's `README` when it's a library written in Java (versus one written in Kotlin, which implies it requires Kotlin). See, for example, the docs for [Android-State / State Saver](https://github.com/evernote/android-state) and 
[Icepick](https://github.com/frankiesardo/icepick). `Android-State` was even written for Kotlin compatibility but still primarily uses Java in its examples and only uses Kotlin examples in places where properties (rather than fields) are involved. It also might be a bit strange to include Kotlin versions for some, but not all, of the examples in the `README`. So are you cool with just removing that part of the change for now?

(I'll note that maybe the best option here would be to have some way to "toggle" between the examples in Java and Kotlin like the new Android docs do, but unfortunately I'm not aware of a way to do such a thing in Markdown at the moment. I'll investigate a bit more, though.)

**Comment by JEuler** [comment]
Hm. I think you are right! Never thought about this from that perspective ("paying respect to library main language"). So, I'm cool. Can remove it for you. 

About the toggle - no, there are no such nice toggle in markdown as I know. https://gist.github.com/Ashton-W/864d09d71f7cb8e5b47ebaac02b98455 - that's how it can be done, but this is not cool as in android docs.

**Comment by byencho** [comment]
Yeah I was just discussing with someone if there was a way to use that kind of "collapsible" control as the tab-like "toggle" I'd like to see but we haven't come up with a way to do it yet. Because ideally we'd always want one to be expanded, rather than having both be collapsible.

**Comment by JEuler** [comment]
I've asked them about this capability (https://twitter.com/ITerhin/status/1110226548655882242). But I think it is impossible. So, I have cleaned up the Kotlin example.

**Comment by byencho** [comment]
Sounds good, thanks!

**Pull request merged**